src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
it task description there is:
All instances of the Animal class must have properties health and a name. Health is 100 by default.

I think it means: health is 100 by default, but can be different
please make constructor take also health and set it to 100 if it is not passed